tempCodeRunnerFile.mjs

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
cos'è questo file?

Reply:
È quello che mi crea VA in automatico per non so che cosa, prossima volta mi ricorderò di non pusharlo, grazie mille